Caboodle/Clipboard/Clipboard.android.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class Clipboard
+    {
+        static ClipboardManager clipboardManager;
```

Comment:
I am now kind of leaning towards not having this around being static and just having it part of the method. I think it can just query the current one. This will enable us to not have the object around in memory.